src/css/main.less

```diff
@@ -26,6 +26,7 @@
 @import "components/filter-box.less";
 @import "components/loading-bar.less";
 @import "components/health-bar.less";
+
```

Comment:
But @aldipower this empty line will not show up in the compiled css :-) I'll remove it anyway.